third_party/move/move-vm/runtime/src/config.rs

```diff
@@ -1,55 +1,37 @@
 // Copyright (c) The Move Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use move_binary_format::{
-    deserializer::DeserializerConfig,
-    file_format_common::{IDENTIFIER_SIZE_MAX, VERSION_MAX},
-};
-use move_bytecode_verifier::VerifierConfig;
 use serde::Serialize;
 
 pub const DEFAULT_MAX_VALUE_NEST_DEPTH: u64 = 128;
 
 /// Dynamic config options for the Move VM.
 #[derive(Clone, Serialize)]
 pub struct VMConfig {
-    pub verifier_config: VerifierConfig,
```

Comment:
Whhy move out the verifier_config and deserilaizer config? Seems that they should go together nicely here?

Reply:
Rationale is that verification + deserialization configs are actually not needed in theVM once we have new loader? In any case, let me roll that back - we can always do this later if we need to, because as you said, keeping them in one place is nice and clean as well.

Reply:
Placed back!